packages/luna/src/components/Chat/SendInput/SendInput.tsx

```diff
@@ -26,8 +50,8 @@ export const SendInput: React.FC<{
   );
 
   useEffect(() => {
-    document.addEventListener("keypress", handleSubmitByEnter);
-    return () => document.removeEventListener("keypress", handleSubmitByEnter);
+    document.addEventListener("keydown", handleSubmitByEnter);
```

Comment:
The `handleSubmitByEnter` accepts the entire `KeyboardEvent` as a param. This make the name not accurate. Submitting by enter is an internal details. We can call the function on the name of the event `onKeydown`. 

```suggestion
    document.addEventListener("keydown", onKeydown);
```